code/addons/test/src/Panel.tsx

```diff
@@ -236,13 +251,33 @@ export const Panel = memo<{ storyId: string }>(function PanelMemoized({ storyId
     !!unhandledErrors ||
     interactions.some((v) => v.status === CallStates.ERROR);
 
+  const storyStatus = storyStatuses[storyId]?.[TEST_PROVIDER_ID];
+
+  const browserTestStatus = React.useMemo<CallStates | null>(() => {
+    if (!isPlaying && (interactions.length > 0 || hasException)) {
+      return hasException ? CallStates.ERROR : CallStates.DONE;
+    }
+    return isPlaying ? CallStates.ACTIVE : null;
+  }, [isPlaying, interactions, hasException]);
+
+  const hasResultMismatch = React.useMemo(() => {
+    return (
+      browserTestStatus !== null &&
+      browserTestStatus !== CallStates.ACTIVE &&
+      storyStatus?.status !== undefined &&
+      statusMap[browserTestStatus] !== storyStatus?.status
```

Comment:
```suggestion
      statusMap[browserTestStatus] !== storyStatus.status
```

The explanation mark shouldn't be necessary here anymore.

Reply:
good catch!